openproblems/tasks/spatial_decomposition/datasets/_utils.py

```diff
@@ -0,0 +1,88 @@
+from .._utils import obs_means
+from anndata import AnnData
+
+import numpy as np
+import random
+
+
+def generate_seq(length=14):
```

Comment:
can you explain what does this do?

Reply:
This is to generate 14 length barcodes, assuming slide seq but not needed now I guess.